src/frontend/src/components/authenticateBox.ts

```diff
@@ -683,7 +686,12 @@ const useIdentityFlow = async <T, I>({
   result satisfies { kind: "pin" };
   const { result: pinResult } = result;
 
-  return { newAnchor: false, ...pinResult };
+  if (pinResult.tag === "ok") {
+    // We log in with an existing PIN anchor, meaning it is _not_ a new anchor
+    return { newAnchor: false, ...pinResult };
+  } else {
+    return pinResult;
```

Comment:
Ditto, does this result have `newAnchor` field?

Reply:
it does not!